File: source/examples/Query/String.kt
val projectsQuery = realm.where<Project>()

// Pass Case.INSENSITIVE as the third argument for case insensitivity.
Log.i(TAG, "Projects that start with 'e': "
  + projectsQuery.beginsWith("name", "e", Case.INSENSITIVE).count())

Log.i(TAG, "Projects that contain 'ie': "
  + projectsQuery.contains("name", "ie").count())

File: docs/source/examples/Query/String.cs
bool ignoreCase = true;
Debug.WriteLine("Projects that start with 'e': "
    + projects.Where(p => p.Name.StartsWith("E", ignoreCase, StringComparison.CurrentCulture)));

Debug.WriteLine("Projects that contain 'ie': "
    + projects.Where(p => p.Name.Contains("ie", StringComparison.OrdinalIgnoreCase)));
